### ISSUE TITLE
Integrate a thinning parameter

### ISSUE THREAD

**Issue by shinyStan** [shinyStan/shinyStan#1]
_From @mhandreae on February 18, 2015 16:46_

Obejects can quickly become too large for SHINYstan to handle even on a modern laptop. Could we have a thin parameter integrated into SHINYstan and possibly a warning if the file seems to be excessively large suggesting to use thinning to display the object?

_Copied from original issue: jgabry/SHINYstan#34_

**Comment by shinyStan** [comment]
Yeah the size of the objects is an issue. Before we do this, we should really ask the stan-dev people what they think about thinning before the user has a chance to do diagnostics, etc.

**Comment by GregoryEady** [comment]
I'm not sure if my problem is similarly related to the model object size, but I have a similar problem at the moment such that shinyStan won't load at all, even an hour after starting launch_shinystan() (R is running CPU at 100%, and has been for an hour now). For reference, I saved the model object to a .RDS file, which ends up being 2.5GB (a large country-year panel time-series IRT model). Is there any way I can load this in shinyStan, or might this be a separate problem?

**Comment by jgabry** [comment]
@GregoryEady Sorry I didn't see this sooner. This is the old shinystan repository. I've reposted this question and a response over at the new repo [here](https://github.com/stan-dev/shinystan/issues/13).
